packages/data-service/src/run-command.ts

```diff
@@ -334,7 +334,7 @@ export const runCommand: RunCommand = (
 
   return db.command(
     { ...spec },
-    { readPreference, ...options }
+    { readPreference, enableUtf8Validation: false, ...options }
```

Comment:
This disables utf8 validation for runCommand by default. You can still re-enabe that by specifying it as an option. I had a look through all the places where we use runCommand and it seems to overlap perfectly with all the places where we want to disable the validation. ie. all the places where we get metadata, none of the places where we read/update documents.

Reply:
Yeah ... are you sure this is a good idea? It seems very easy to accidentally enableUtf8Validation when that is _not_ desirable this way

Reply:
OK I removed it from here and then added it to every runCommand individually. Then we can argue about them on a case by case basis :)